### PR TITLE
clarifying a statement

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/15897

### CODE REVIEW

File: observability/observe_environments.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ By default, observability is included with the product installation, but not ena
 
 When the service is enabled, the `observability-endpoint-operator` is automatically deployed to each imported or created cluster. This controller collects the data from {ocp} Prometheus, then sends it to the {product-title-short} hub cluster. 
 
-When observability is enabled in a hub cluster, metrics are collected by handling the hub cluster as a managed cluster called `local-cluster`.
+When observability is enabled in a hub cluster, the hub cluster becomes a managed cluster named `local-cluster`. Metrics are collected by the managed hub cluster.
```
Comment (marcolan018) [inline observability/observe_environments.adoc]: the statement is not correct, "local-cluster" is not created due to observability enablement.

A more exact description is:
If the hub cluster imports itself as "local-cluster", observability will also enabled on it and metrics are collected from hub cluster.

Comment (dockerymick) [reply observability/observe_environments.adoc]: Thanks so much! @marcolan018

File: observability/observe_environments.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ By default, observability is included with the product installation, but not ena
 
 When the service is enabled, the `observability-endpoint-operator` is automatically deployed to each imported or created cluster. This controller collects the data from {ocp} Prometheus, then sends it to the {product-title-short} hub cluster. 
 
-When observability is enabled in a hub cluster, metrics are collected by handling the hub cluster as a managed cluster called `local-cluster`.
+If the hub cluster imports itself as a `local-cluster`, observability is also enabled on it and metrics are collected from hub cluster.
```
Comment (marcolan018) [inline observability/observe_environments.adoc]: looks good to me, but it's better to use "the" local-cluster , not "a" local-cluster?
imports itself as the `local-cluster`

Comment (dockerymick) [reply observability/observe_environments.adoc]: Gotchu, thanks!

**Comment by dockerymick** [comment]
@marcolan018 thanks for your feedback and approval!

**Pull request merged**